site/src/hooks-doc/useTernaryDarkMode/useTernaryDarkMode.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: useTernaryDarkMode
+date: '2022-01-11'
+---
+
+This React Hook offers you an interface to toggle and read the dark theme mode between three values.
+
+The returned value `isDarkMode` is a boolean to let you be able to use with your logic.
+The returned value `ternaryModeCode` is a enum of string of type `"dark" | "system" | "light"`.
+When `ternaryModeCode` is set to `system`, the `isDarkMode` will use system theme, like in iOS and MacOS.
+
```

Comment:
Add a sentence to talk about the new `setTernaryDarkMode` method.

Reply:
added on `b553466`